spi/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/system/SPI.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.spi.system;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Denotes an SPI module.
+ */
+@Target({ElementType.PACKAGE})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface SPI {
```

Comment:
I don't have a strong opinion either way, although I thought it looked a bit strange as "Spi" since it is not a prefix and it abbreviates multiple words. Maybe we should ask what others think as well?

Reply:
OK, based on feedback, changed to @Spi